br/pkg/restore/split/split.go

```diff
@@ -265,7 +268,7 @@ func PaginateScanRegion(
 		return nil
 	}, backoffer)
 
-	return lastRegions, err
+	return lastRegions, retryErr
```

Comment:
```suggestion
	return lastRegions, retryErr
```

should better use last error, or the error stack will be very large.